tests/savepoint/translate/overrides/standard.yaml

```diff
@@ -105,6 +105,38 @@ FVSubgridZ:
   - backend: gtc:cuda
     max_error: 1e-8
 
+GridGrid:
+  - backend: numpy
+    ignore_near_zero_errors:
+      grid: 1e-14
+
+MoreAreas:
+  - backend: numpy
+    max_error: 3e-12
+
+GridAreas:
+  - backend: numpy
+    max_error: 3e-12
+    ignore_near_zero_errors:
+      agrid: 3e-14
+      dxc: 3e-14
+      dyc: 3e-14
+
+AGrid:
+  - backend: numpy
+    max_error: 3e-14
+
+DxDy:
+  - backend: numpy
+    max_error: 3e-14
+
+InitGrid:
+  - backend: numpy
+    max_error: 3e-12
+    ignore_near_zero_errors:
+      gridvar: 3e-14
+      agrid: 3e-14
```

Comment:
Could you put the basic override values in the corresponding Translate classes? I believe this file is meant for overriding the "normal" thresholds for each translate class, not for setting these "normal" values.